src/components/Download/StartProcessingForm/TermsOfUseCheckBox.js

```diff
@@ -0,0 +1,41 @@
+import { Box, CheckBox, Text } from 'grommet'
+import { links } from 'config'
+import { Anchor } from 'components/shared/Anchor'
+import { InlineMessage } from 'components/shared/InlineMessage'
+
+export const TermsOfUseCheckBox = ({ error, touched, value, handleChange }) => {
+  return (
+    <Box pad={{ top: 'small' }} style={{ position: 'relative' }}>
+      {error && touched && (
+        <Box animation={{ type: 'fadeIn', duration: 300 }}>
+          <InlineMessage
+            color="error"
+            height="16px"
+            justify="center"
+            label={error}
+            iconSize="small"
+            style={{ position: 'absolute', top: '2px' }}
+          />
+        </Box>
+      )}
+      <CheckBox
```

Comment:
Continuing the thread from here: https://github.com/AlexsLemonade/refinebio-web/pull/202#discussion_r1364275153

Yeah I agree that we should wrap checkbox with error handling to match how we handle it in `TextBox`. Good point.